main.go

```diff
@@ -4,13 +4,17 @@ import (
 	"context"
 	"fmt"
 
-	"github.com/billz-2/ofd_connector/pkg/httpclient"
+	"github.com/billz-2/ofd_connector/internal/httpclient"
 )
 
 // OfdConnector interface defines the contract for OFD service operations
 type OfdConnector interface {
 	FiscalDriveList(context.Context) ([]FiscalDriveReaderInfo, error)
 	SetFactoryID(factoryID string)
+
+	OpenZreport(ctx context.Context, createdTime string) error
```

Comment:
i would use here interface segregation and composition for small and supportable services.
i.e. would add `ZReportI` as an interface with requesting `ZReport` methods. AS far as i remember there is list of OFD methods more than 10 if we will add them all in one and implement in the same service it seems that a service will be huge.